scripts/tasks/lint-imports.js

```diff
@@ -311,6 +320,7 @@ function lintImports() {
   function reportFilePathErrors(importErrors) {
     /** @type {{ [k in keyof ImportErrors]: string }} */
     const errorMessages = {
+      pathAbsolute: 'files are using absolute imports. ' + 'Please update the following imports to use relative paths instead:',
```

Comment:
nit: combine these strings
```suggestion
      pathAbsolute: 'files are using absolute imports. Please update the following imports to use relative paths instead:',
```